### ISSUE TITLE
Empty DataTable message uses presentational attributes

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1132]
**Describe the bug**
When a DataTable is created using the `data-empty-message` attribute, the resulting message uses `valign="top"` which throws a SiteImprove error.

**To Reproduce**
```
{{ 
    html.datatable({
        'data-empty-table': 'There are currently no Arnie films to display',
        'data-order': '[[ 3, "desc" ]]',
        'data': data
    }) 
}}
```

**Expected behavior**
The `valign` attribute should not be present and any related styles should be moved to CSS.

**Screenshots**
![Screenshot 2020-02-06 at 10 39 59](https://user-images.githubusercontent.com/18653/73929816-6a788400-48cd-11ea-8795-3d64766406b6.png)

**Comment by jamesjacobs** [comment]
Fixed by #1133
